vercel/assets/dashboards/vercel_overview.json

```diff
@@ -158,13 +110,61 @@
       },
       "layout": { "x": 8, "y": 0, "width": 4, "height": 5 }
     },
+    {
+      "id": 6397985032849882,
+      "definition": {
+        "type": "note",
+        "content": "## [Logs](https://docs.datadoghq.com/integrations/vercel/)\n\nTo get visibility into application logs from Vercel, configure the [Vercel logs integration](https://docs.datadoghq.com/integrations/vercel/).",
```

Comment:
I *think* this is going to have the same region issue. I *think* the fix is to use `{{< region-param key="dd_site_name" >}}` in place of `datadoghq.com`, but I don't have a way to test that yet.

My concern is someone is logged into an EU instance, they click this and it sends them to the NA docs, which then link them to the NA version of the setup if they click the "configure" button from the docs.

Reply:
Oh yeah, here I think we can actually just use a relative URL since this is not being replicated in the docs. Will update

Reply:
Oh wait we are using the relative URL in the link to the Vercel UI, these links are to the docs site so I think it's not a problem for them to be absolute